feathr_project/feathr/registry/_feathr_registry_client.py

```diff
@@ -85,8 +84,10 @@ def __init__(self, project_name: str, endpoint: str, project_tags: Dict[str, str
         self.project_name = project_name
         self.project_tags = project_tags
         self.endpoint = endpoint
+        # TODO: expand to more credential provider
+        # If FEATHR_SANDBOX is set in the environment variable, don't do auth
```

Comment:
If FATHER_SANDBOX don't do auth, RBAC related content could also be removed, e.g. in start_local.sh.

Reply:
I think we potentially can add RBAC back in the future?